City-of-Helsinki/drupal-helfi-sote
# UHF-5996: Alter unit contact and accessibility

# [UHF-5996](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5996)
Made the requested changes to the _Unit contact card_ and _Unit accessibility information_ paragraphs.

## What was done
* Added ability to choose which picture – TPR or overridden – is used with _Unit contact card_.
* Added the _Unit contact card_ paragaph also to the basic page's upper content area.
* Added the _Unit accessibility information_ paragraph to the same content areas as the _Unit contact card_ paragaph.

## How to install
* Make sure your instance is up and running on correct branch, e.g:
  * `git pull origin UHF-5996-alter-unit-contact-and-accessibility`
  * `make fresh`

## How to test
* Check that _Unit contact card_ and _Unit accessibility information_ paragraphs can be added to landing page, basic page (both content areas), TPR unit (both content areas) and TPR service (both content areas).
* Check that when adding the _Unit contact card_, there is an option to prefer override picture: when selected, the override picture is shown if available; when not selected, the TPR picture is shown.


## Designers review
